.github/workflows/build_windows_exe.yml

```diff
@@ -0,0 +1,20 @@
+name: Build_Windows_exe
+
+on:
+  push:
+    tags:
+    - "*"
+
+jobs:
+  Build_Windows_exe:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - run: |
+        python -m pip install pyinstaller
+        python setup.py clean bundle
```

Comment:
Please split this up into two `run` blocks. This way it will be easier to spot where things went wrong when the build fails.